### PR TITLE
Update es.ts

### PR DESCRIPTION
**Pull request by franscal7**
missing translations are added

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are updated
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by joebuono — APPROVED** [review]
Thank you for adding these translations!

File: packages/ui/src/i18n/dictionaries/authenticator/es.ts [hunk 1]
```diff
@@ -3,31 +3,49 @@ export const esDict = {
...
   'Resend a Code': 'Reenviar un código',
+  'Reset your password': 'Restablecer su contraseña',
+  'Reset your Password': 'Restablecer su Contraseña',
   'Resend Code': 'Reenviar código',
   'Send Code': 'Enviar código',
+  Sending: 'Enviando',
+  'Setup TOTP': 'Configurar TOTP',
+  'Show password': 'Mostrar contraseña',
   'Sign In': 'Iniciar sesión',
+  'Sign In': 'Iniciar Sesión',
```
Comment (reesscot) [inline packages/ui/src/i18n/dictionaries/authenticator/es.ts:40]: This appears to be a duplicate of the translation above. Do we need both uppercase `Sign In` and lowercase `Sign in`?

Comment (zchenwei) [reply packages/ui/src/i18n/dictionaries/authenticator/es.ts:40]: +1

Comment (franscal7) [reply packages/ui/src/i18n/dictionaries/authenticator/es.ts:40]: I base on the en.ts file

Comment (reesscot) [reply packages/ui/src/i18n/dictionaries/authenticator/es.ts:40]: Yes, but this is adding a second `Sign In` key. Did you mean to add a `Sign in` key here instead?

**Review by reesscot — CHANGES_REQUESTED** [review]
Looks good, we just need to fix the duplicate `Sign In` key

**Comment by reesscot** [comment]
See https://lgtm.com/projects/g/aws-amplify/amplify-ui/snapshot/14f17effd8f2914977fd2a5b55edb3eea973a492/files/packages/ui/src/i18n/dictionaries/authenticator/es.ts?sort=name&dir=ASC&mode=heatmap#xf8a4113e8f2e0722:1

File: packages/ui/src/i18n/dictionaries/authenticator/es.ts [hunk 2]
```diff
@@ -3,31 +3,49 @@ export const esDict = {
...
+  'Creating Account': 'Creando cuenta',
   Email: 'Email',
   'Forgot Password': 'Olvidé mi contraseña',
   'Incorrect username or password': 'Nombre de usuario o contraseña incorrecta',
   'Invalid password format': 'Formato de contraseña inválido',
   'Invalid phone number format': 'Formato de número de teléfono inválido',
+  'Forgot your password?': '¿Olvidó su contraseña?',
+  'Hide password': 'Ocultar contraseña',
   'Loading...': 'Cargando...',
+  Username: 'Nombre de usuario',
```
Comment (reesscot) [inline packages/ui/src/i18n/dictionaries/authenticator/es.ts:26]: Please remove this duplicate entry, Username is already on line 53
```suggestion
```

**Review by reesscot — APPROVED** [review]
LGTM

**Review by zchenwei — APPROVED** [review]
LGTM

**Comment by reesscot** [comment]
@franscal7 Can you please merge main into your branch so we can run the integration tests? There were recent changes on main that are needed in the test suite.

**Pull request merged**